Source/NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -524,7 +522,7 @@ public final class NVActivityIndicatorView: UIView {
 
     // swiftlint:disable:next identifier_name
     func _setTypeName(_ typeName: String) {
-        for item in NVActivityIndicatorType.allTypes {
+        for item in NVActivityIndicatorType.allCases {
```

Comment:
We also need to update this in the example project.

Reply:
I changed the example project too.